tests/jax/components/training/losses_test.py

```diff
@@ -0,0 +1,192 @@
+from types import SimpleNamespace
+from typing import Callable, Tuple
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from mava.components.jax.training.losses import (
+    MAPGTrustRegionClippingLossConfig,
+    MAPGWithTrustRegionClippingLoss,
+)
+from mava.systems.jax.trainer import Trainer
+
+
+class MockNet:
+    """Creates a mock network for loss function"""
+
+    def apply(
+        parameters: jnp.array, observation: jnp.array
+    ) -> Tuple[jnp.ndarray, jnp.ndarray]:
+        """Mock function to apply the network to training data"""
+        return observation, observation
```

Comment:
`return parameters, observation` surely?

Reply:
But can't you just assert for the value of the parameters in the output?

Reply:
It's actually supposed to be a dictionary. It's been changed to use the parameters value.Thanks for catching that!